### PR TITLE
add enrichment to proj settings

### PR DESCRIPTION
**Pull request by dchhabda**

### CODE REVIEW

File: templates/projects/enrichment.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+{% assets output="css/gen/enrichment.min.css",
+          "css/jsgrid.min.css",
+          "css/jsgrid-theme.min.css"
+%}
+    <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">
+{% endassets %}
+
+{% assets filters="jsmin", output="js/gen/enrichment.min.js",
+          "js/vendor/jquery-2.2.3.js",
```
Comment (fiorda) [inline templates/projects/enrichment.html]: why jquery 2.2.3?

File: templates/projects/enrichment.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{% extends "/base.html" %}
```
Comment (fiorda) [inline templates/projects/enrichment.html]: Do you need to extend `/base.html` or `/projects/base.html`?

File: templates/projects/settings.html [hunk 3]
```diff
@@ -50,4 +50,14 @@
 {% endif %}
 
 {% endif %}
+
+<div class="row">
+    <div id="import_tasks" class="col-md-6">
+        {{ render_project_card_option(project, upload_method, title=_('Delete the project'), explanation=_('Delete the project, tasks, task runs and answers.'), link=url_for('project.delete', short_name=project.short_name), link_action_text=_('Delete'), icon='trash')}}
```
Comment (fiorda) [inline templates/projects/settings.html]: Do you have two delete buttons?

File: templates/projects/js/enrichment.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            type: 'POST',
+            dataType: 'json',
+            contentType: 'application/json',            
+            data: JSON.stringify(data),
+            success: function(response) {
+                var url = $SCRIPT_ROOT + "/project/" + short_name + "/settings";
+                window.location.href = url
+            },
+            error: function(error) {
+                console.log(error);
```
Comment (fiorda) [inline templates/projects/js/enrichment.js]: We can display some error here

File: templates/projects/js/enrichment.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<script>
+var csrftoken = "{{ csrf_token() }}";
+    $.ajaxSetup({
+    beforeSend: function(xhr, settings) {
+        if (!/^(GET|HEAD|OPTIONS|TRACE)$/i.test(settings.type)) {
+            xhr.setRequestHeader("X-CSRFToken", csrftoken);
+        }
+    }
+    });
+    $SCRIPT_ROOT = {{ request.script_root|tojson|safe }};
```
Comment (fiorda) [inline templates/projects/js/enrichment.js]: can you use `window.location`?

File: templates/projects/enrichment.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+{% endassets %}
+
+<script>
+    var enrich_data = {{ enrichments | safe }};
+    var enrichment_types = {{ enrichment_types | safe }};
+    
+    var enrich_types = [];
+    for(var et in enrichment_types) enrich_types.push({ Name: et, Value: et });
+    for (var first_etype in enrichment_types) break;
+    enrichment_subtypes = enrichment_types[first_etype];
```
Comment (fiorda) [inline templates/projects/enrichment.html]: missing `var`

File: templates/projects/enrichment.html [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+        onItemUpdated: function(args) {
+            document.getElementById("saveenrich").disabled = false;
+        },
+        onItemDeleted: function(args) {
+            document.getElementById("saveenrich").disabled = false;
+        },
+        data: enrich_data,
+        fields: [
+            { title: "Type", name: 'type', type: "select", items: enrich_types, valueField: "Value", textField: "Name", validate: "required" },
+            { title: "Subtype", name: 'sub_type', type: "select", items: enrich_subtypes, valueField: "Value", textField: "Name", validate: "required" },
```
Comment (fiorda) [inline templates/projects/enrichment.html]: This is not dynamic, right?

Comment (dchhabda) [reply templates/projects/enrichment.html]: yes

**Pull request merged**